kafka-elasticsearch/src/main/java/com/elte/elasticSearchConsumer/elasticsearch.properties

```diff
@@ -0,0 +1,4 @@
+#i'll hide this part for now and will use it during demonstration
+hostname=anomalydetection-tes-5738607777.eu-central-1.bonsaisearch.net
+username=1y3i5gwyif
+password=m8llvspz5k
```

Comment:
Should not credentials be hidden? I mean not pushing to repository

Reply:
Yes. I added this file to .gitignore, but missed to notice that it still got committed. I'll remove it from the repo and submit another request